src/main/resources/index.jelly

```diff
@@ -1,4 +1,5 @@
 <?jelly escape-by-default='true'?>
 <div>
-    Delphix
-</div>
+     Allow Jenkins Users to create and delete Delphix Objects.
```

Comment:
Please update: 
The Delphix Plugin enables application teams to integrate the power of Delphix's DevOps Data Platform within their automated Jenkins Pipelines to enhance DevOps Test Data Management practices.